SettingsList.py

```diff
@@ -619,7 +619,8 @@ def __init__(self, name, gui_text, min, max, default, step=1,
                     expire. You can get extra time to show the Eye Drops if
                     you warp immediately upon receiving them. If you don't
                     have many hearts, you may have to reset the heat timer
-                    by quickly dipping in and out of Darunia's chamber.
+                    by quickly dipping in and out of Darunia's chamber or 
+                    quickly equipping and unequipping the Goron Tunic.
```

Comment:
While technically that does reset the heat timer, we'd just assume they can wear the Goron Tunic and have no heat timer. Probably not worth suggesting to do this...

Reply:
So the trick is only relevant when the Goron City entrance goes to Goron City, so the timer reset in Darunia's Chamber is always going to apply. But since City->Trail through Crater has a tunic/FTR check now, if FTR is off that trek is going to expect Goron Tunic even though wearing it is going to cause you to fail the trick. So I guess in theory, the quick equip/unequip is what's being expected there?

Reply:
r0bd0g has it about right (which makes sense, since he suggested this additional line in the first place). Because city->trail can now logically require goron tunic (if FTR is off), there needs to be something explaining how to do the trick when simply dipping city is not expected to be enough.

Reply:
Ah, because you have to have tunic off to use the trick.